diff --git a/node_modules/sockjs-client/dist/sockjs.js b/node_modules/sockjs-client/dist/sockjs.js
index ee4a359..79e63b2 100644
--- a/node_modules/sockjs-client/dist/sockjs.js
+++ b/node_modules/sockjs-client/dist/sockjs.js
@@ -3282,7 +3282,7 @@ var onUnload = {}
 module.exports = {
   attachEvent: function(event, listener) {
     if (typeof global.addEventListener !== 'undefined') {
-      global.addEventListener(event, listener, false);
+      // global.addEventListener(event, listener, false);
     } else if (global.document && global.attachEvent) {
       // IE quirks.
       // According to: http://stevesouders.com/misc/test-postmessage.php
@@ -3295,7 +3295,7 @@ module.exports = {
 
 , detachEvent: function(event, listener) {
     if (typeof global.addEventListener !== 'undefined') {
-      global.removeEventListener(event, listener, false);
+      // global.removeEventListener(event, listener, false);
     } else if (global.document && global.detachEvent) {
       global.document.detachEvent('on' + event, listener);
       global.detachEvent('on' + event, listener);
